config-controller/pkg/client/exec-sshd.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
> Should these jsons be checked in?

I am using these two policies for unit tests via `embed.FS`, and I wasn't sure how else to refer to policy files.  I picked these two default policies at random.

> Also the locations should be consistent for both jsons if we intend to check them in?

They look to be in the same folder.  Is that that you meant?

Reply:
Yeah I'd "craft my own" instead of using default policies for less confusion in the code base/readability/future novice going wth is happening :-), and yes they didn't seem to show up in the same folder as I had expected them too (maybe the left hand side git source tree had some suspicious visualization)

Reply:
I addressed this in 8062b238a066904787e637003d502e3eccb54ca5, but please double check to make sure it looks like you expect.